components/svg/UniqueIconComposer.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import { IconComposer } from "@rubin-epo/epo-react-lib";
+import UniqueIcons from "./unique";
+
+/** adds the unique icons for rubin-obs-client to the icons
+ *  already defined in @rubin-epo/epo-react-lib
+ */
+const UniqueIconComposer = (props) => (
```

Comment:
simple reimplementation of the IconComposer that extends it with new icons and passes on other props.